packages/rum/src/constants.ts

```diff
@@ -1,22 +1,15 @@
 export const NodePrivacyLevel = {
```

Comment:
Is it possible to reuse the `InitialPrivacyLevel` constant here and extend it like?
```ts
export const NodePrivacyLevel = {
  ... InitialPrivacyLevel,
  IGNORE: 'ignore',
  HIDDEN: 'hidden',
}
```

Reply:
It is!